superset/db_engine_specs/presto.py

```diff
@@ -174,7 +174,9 @@ def get_view_names(
         return [row[0] for row in results]
 
     @classmethod
-    def _create_column_info(cls, name: str, data_type: str) -> Dict[str, Any]:
+    def _create_column_info(
+        cls, name: str, data_type: types.TypeEngine
+    ) -> Dict[str, Any]:
```

Comment:
Not sure why MyPy hadn't picked up on `data_type` being of type `TypeEngine`, not `str`.